src/features/data-streams/common/gettingStartedHardhat.mdx

```diff
@@ -0,0 +1,213 @@
+import { Aside, CodeSample, CopyText } from "@components"
+import DataStreams from "@features/data-streams/common/DataStreams.astro"
+
+This guide shows you how to read data from a Data Streams feed, verify the answer onchain, and store it. This example uses a [Chainlink Automation Log Trigger](/chainlink-automation/guides/log-trigger) to check for events that require data. The flow follows this sequence:
```

Comment:
```suggestion
This guide shows you how to read data from a Data Streams feed, verify the answer onchain, and store it. This CLI guide uses the [Hardhat Framework](https://hardhat.org/) so you can complete these steps using terminal commands rather than the web-based Remix IDE. If you prefer Remix or are unfamiliar with how to run terminal commands, read the [Getting Started - Remix IDE](data-streams/getting-started) guide instead.

This example uses a [Chainlink Automation Log Trigger](/chainlink-automation/guides/log-trigger) to check for events that require data. The flow follows this sequence:
```

Feel free to edit this suggestion if it does not read clearly.